docs/C++/home.md

```diff
@@ -0,0 +1,8 @@
+---
+id: C++-home
+title: C++ Tutorial Overview
+sidebar_label: Overview
+sidebar_position: 
```

Comment:
`sidebar_position` value is required if you want to use then can't excepted null so, put numerical value